_sass/_progress.scss

```diff
@@ -24,6 +51,18 @@
     padding: 14px 3.5% 0px 3.5%;
     position: relative;
     z-index: 99;
+    @include breakpoint-large{
+      padding-bottom: 1em;
+    }
+
+    @include breakpoint-medium {
+      padding-bottom: 0em;
+
```

Comment:
Since we're a larger team, let's try to be careful about white space.  Can you clean up any empty lines that might be left?